record-builder-processor/src/main/java/io/soabase/recordbuilder/processor/InternalRecordBuilderProcessor.java

```diff
@@ -72,6 +73,9 @@
                 .addTypeVariables(typeVariables);
         addVisibility(recordActualPackage.equals(packageName), record.getModifiers());
         addWithNestedClass();
+        if (!metaData.beanClassName().isEmpty()) {
```

Comment:
This should also check that the other required options are set. I did a quick test where I removed `setterPrefix = "set", getterPrefix = "get", booleanPrefix = "is", ` and you then get compilation failure. Let's either just ignore this setting if the others aren't sent our output a message.

Reply:
Really? I did the same in CustomMethodNames.class, and it built just fine with an empty bean interface:
`
    /**
     * Add getters to {@code CustomMethodNames}
     */
    @Generated("io.soabase.recordbuilder.core.RecordBuilder")
    public interface Bean {
    }
`

Reply:
I used this:

```
@RecordBuilder
@RecordBuilder.Options(beanClassName = "Bean")
public record CustomMethodNames(
    int theValue,
    List<Integer> theList,
    boolean theBoolean) implements Bean {
}
```

I then get:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:testCompile (default-testCompile) on project record-builder-test: Compilation failure: Compilation failure: 
[ERROR] /Users/jordanzimmerman/dev/oss/soabase/record-builder/record-builder-test/src/test/java/io/soabase/recordbuilder/test/TestCustomMethodNames.java:[29,9] cannot find symbol
[ERROR]   symbol:   method setTheValue(int)
[ERROR]   location: class io.soabase.recordbuilder.test.CustomMethodNamesBuilder
[ERROR] /Users/jordanzimmerman/dev/oss/soabase/record-builder/record-builder-test/src/test/java/io/soabase/recordbuilder/test/TestCustomMethodNames.java:[41,13] cannot find symbol
[ERROR]   symbol:   method setTheValue(int)
[ERROR]   location: class io.soabase.recordbuilder.test.CustomMethodNamesBuilder
[ERROR] /Users/jordanzimmerman/dev/oss/soabase/record-builder/record-builder-test/src/test/java/io/soabase/recordbuilder/test/TestCustomMethodNames.java:[53,24] cannot find symbol
[ERROR]   symbol:   method getTheValue()
[ERROR]   location: variable obj of type io.soabase.recordbuilder.test.CustomMethodNames
[ERROR] /Users/jordanzimmerman/dev/oss/soabase/record-builder/record-builder-test/src/test/java/io/soabase/recordbuilder/test/TestCustomMethodNames.java:[54,33] cannot find symbol
[ERROR]   symbol:   method getTheList()
[ERROR]   location: variable obj of type io.soabase.recordbuilder.test.CustomMethodNames
[ERROR] /Users/jordanzimmerman/dev/oss/soabase/record-builder/record-builder-test/src/test/java/io/soabase/recordbuilder/test/TestCustomMethodNames.java:[55,19] cannot find symbol
[ERROR]   symbol:   method isTheBoolean()
[ERROR]   location: variable obj of type io.soabase.recordbuilder.test.CustomMethodNames
[ERROR] -> [Help 1]
[ERROR] 
[ERR
```

Reply:
Oh wait - those errors are from the test - never mind

Reply:
However, you do end up with an empty `Bean` class. Do we want that?

Reply:
Well, while completely useless, you *did* enable the feature to generate the interface, so I would claim it's expected behavior. I can of course add a clause to the if to make it go away in this case, but there's still two corner cases that would result in the same empty interface:

Only getter prefix is enabled, but only boolean fields in record
Only boolean prefix is enabled, but no booleans in record

These two would require me to scan the record for these fields, and at that point I feel I'm making a non-negligible amount of code to check for very little gain. Let me know if you still want me to amend that if and just ignore these corner cases

Reply:
> Well, while completely useless, you did enable the feature to generate the interface

Sure - that makes sense.